src/components/RegionLinks/RegionLinks.test.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { axe, render, screen } from 'testUtils';
+import RegionLinks from './';
+import { fireEvent } from '@testing-library/react';
+
+const onClickMock = jest.fn();
+const props = {
+  title: 'Victoria',
+  regions: [
+    { href: '/australia/vic/melbourne', name: 'Melbourne' },
+    {
+      href: '/search/list?location=mornington peninsula',
+      name: 'Mornington Peninsula',
+    },
+    { href: '/search/list?location=daylesford', name: 'Daylesford' },
+  ],
+  onClick: onClickMock,
+};
+
+beforeEach(() => {
+  jest.clearAllMocks();
+});
+
+test('renders correctly with title', () => {
+  render(<RegionLinks {...props} />);
+  expect(screen.getByRole('heading', { level: 3 })).toBeInTheDocument();
+});
+
+test('renders correctly with no title', () => {
+  render(<RegionLinks {...props} title={undefined} />);
+  expect(screen.queryByRole('heading', { level: 3 })).not.toBeInTheDocument();
+});
+
+test('shows the heading', () => {
+  render(<RegionLinks {...props} />);
+  expect(
+    screen.getByRole('heading', { level: 3, name: 'Victoria' }),
+  ).toBeInTheDocument();
+});
+
+test('shows the regions', () => {
+  render(<RegionLinks {...props} />);
+  expect(screen.getByText('Melbourne')).toHaveAttribute(
```

Comment:
```suggestion
  expect(screen.getByRole('link', { name: 'Melbourne' })).toHaveAttribute(
```
Same for the tests here